Extensions/TextObject/Extension.cpp

```diff
@@ -364,6 +428,65 @@ void DeclareTextObjectExtension(gd::PlatformExtension& extension) {
       .SetManipulatedType("number")
       .SetIncludeFile("TextObject/TextObject.h");
 
+  obj.AddCondition("GetPadding",
+                   _("Padding"),
+                   _("Compare the number of pixels around a text object. If you text gets cropped due to shadow effects or an outline raise this value."),
+                   _("The padding of _PARAM0_ is _PARAM1__PARAM2_"),
+                   _("Style"),
+                   "res/conditions/textPadding24.png",
+                   "res/conditions/textPadding.png")
+
+      .AddParameter("object", _("Object"), "Text")
+      .AddParameter("relationalOperator", _("Sign of the test"))
+      .AddParameter("expression", _("Value to test"))
+      .SetFunctionName("GetPadding")
+      .SetManipulatedType("number")
+      .SetIncludeFile("TextObject/TextObject.h");
+
+  obj.AddAction("SetPadding",
+                _("Padding"),
+                _("Set the number of pixels around a text object. If you text gets cropped due to shadow effects or an outline raise this value."),
```

Comment:
I would rather say:

`If the shadow or the outline around the text are getting cropped, raise this value.`